src/components/withToggleVisibilityView.tsx

```diff
@@ -5,12 +5,12 @@ import type {SetOptional} from 'type-fest';
 import useThemeStyles from '@hooks/useThemeStyles';
 import getComponentDisplayName from '@libs/getComponentDisplayName';
 
-type ToggleVisibilityViewProps = {
+type WithToggleVisibilityViewProps = {
     /** Whether the content is visible. */
-    isVisible: boolean;
+    isVisible?: boolean;
```

Comment:
Yes, it's used in `BaseValidateCodeForm.tsx` and if not optional, it will cause type errors in `ValidateCodeForm/` as the component expects `isVisible` prop which actually comes from a HOC.

Reply:
- BaseValidateCodeForm expects `isVisible` to have type `boolean` since we always set it in withToggleVisibilityView
- ValidateCodeForm expects  `isVisible` to have type `boolean | undefined`


So I think we should revert the change here to make the prop required and modify ValidateCodeFormProps to make it accept an optional isVisible prop.



___

We pass isVisible to ValidateCodeForm in SignInPage
https://github.com/Expensify/App/blob/a839310515fc175700e3b1d0d2a3a40d3862d2ac/src/pages/signin/SignInPage.js#L282

Reply:
Updated!